src/server/dfly_main.cc

```diff
@@ -611,9 +611,11 @@ void sigill_hdlr(int signo) {
 }
 
 void PrintBasicUsageInfo() {
+  auto ends_with_slash = [](const string_view v) { return v.back() == '/'; };
   std::cout << "* Logs will be written to the first available of the following paths:\n";
   for (const auto& dir : google::GetLoggingDirectories()) {
-    std::cout << dir << "dragonfly.*\n";
+    const string maybe_slash = ends_with_slash(dir) ? "" : "/";
```

Comment:
I'd use `string_view` instead of a string. It's true that both options will fit way below small-string optimization, but still

Reply:
wouldn't that cause the temporary to dangle? Otherwise it should be `const &` which can then extend the lifetime? Am I missing something?

Reply:
literal strings (like `"/"`) don't have any lifetime, so it's safe to point `string_view`s to them